### ISSUE TITLE
Multisite for compatibility

### ISSUE THREAD

**Issue by carl-alberto** [pantheon-systems/example-wordpress-composer#67]
Would this be worthy to mention that this example is not multisite compatible out of the box in Pantheon? Would it also worth exploring the possibility of creating an example of this which is multisite and compatible in the platform?

It might need some additional plugin https://github.com/roots/bedrock/issues/250 and some nginx mods to make it work

**Comment by ataylorme** [comment]
You are correct that this project, like any other WordPress self-serve site on Pantheon, does not support multisite. We could explicitly reference that in the README for this project.

WordPress site networks (multisite) are only available to contract customers and must be created by Pantheon (see [our site WordPress network doc](https://pantheon.io/docs/guides/multisite/#request-a-wordpress-site-network)).

If someone wanted to use Composer and nested docroot with multisite like this example demonstrates that is something we could potentially help with during on-boarding and creation of the multisite.

**Comment by wss-chadical** [comment]
@ataylorme Thx for the response Andrew. In fact, we are currently working with Scott Page on enterprise pricing. We will probably get into contract Dec/Jan. Until we are using Basic with full-on Lando+Pantheon+Wordpress+Composer setup.

So, "If someone wanted to use Composer and nested docroot with multisite like this example demonstrates that is something we could potentially help with during on-boarding and creation of the multisite." is exactly us! Looking forward to working with Pantheon.

**Comment by carl-alberto** [comment]
Thanks for the clarification @ataylorme, hopefully we can add these clarifications in this doc and other docs that we have mentions regarding multisite so users are better informed
